frontend/src/Editor/Components/Table/columns/actions.jsx

```diff
@@ -17,6 +17,15 @@ const generateActionsData = ({ actions, columnSizes, defaultColumn, fireEvent, s
                 <button
                   key={action.name}
                   className="btn btn-sm m-1 btn-light"
+                  disabled={
+                    action?.buttonState !== undefined
+                      ? typeof resolveReferences(action.buttonState, currentState) == 'string'
```

Comment:
This resolution call looks unnecessary 🤔 

Generally, there are too many resolutions here, why not have a single one?